### PR TITLE
0412

### PR DESCRIPTION
**Pull request by syueer**
- [x] Fizz
- [x] buzz

### CODE REVIEW

File: Typescript/0049.Group_Anagrams/0049Solution.ts [hunk 1]
```diff
@@ -0,0 +1,14 @@
+function groupAnagrams(strs: string[]): string[][] {
+  let obj: Record<string, string[]> = {};
+  for (let i = 0; i < strs.length; i++) {
+    let key: string = strs[i].split('').sort().join('');
+    console.log(typeof key);
```
Comment (Allen-Ning) [inline Typescript/0049.Group_Anagrams/0049Solution.ts]: please remove console log

File: Typescript/0049.Group_Anagrams/0049Solution.ts [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+    if (obj === {} || !obj[key]) {
```
Comment (Allen-Ning) [inline Typescript/0049.Group_Anagrams/0049Solution.ts]:
```
if (key in object) { xxx }
```

File: Typescript/0412.Fizz_Buzz/0412Solution.ts [hunk 3]
```diff
@@ -0,0 +1,15 @@
+function fizzBuzz(n: number): string[] {
+  let array = [];
+  for (let i = 0; i < n; i++) {
```
Comment (Allen-Ning) [inline Typescript/0412.Fizz_Buzz/0412Solution.ts]: `for (let i = 1; i <= n; i++) {  xxx }`

File: Typescript/0412.Fizz_Buzz/0412Solution.ts [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+    if ((i + 1) % 5 === 0 && (i + 1) % 3 === 0) {
+      array[i] = 'FizzBuzz';
```
Comment (Allen-Ning) [inline Typescript/0412.Fizz_Buzz/0412Solution.ts]: `array.push('FizzBuzz');`

File: Typescript/0049.Group_Anagrams/0049Solution.ts (hunk 2 above)
Comment (syueer) [reply Typescript/0049.Group_Anagrams/0049Solution.ts]: fixed

**Review by Allen-Ning — APPROVED** [review]
lgtm

**Pull request merged**